appveyor-nightly.yml

```diff
@@ -1,64 +1,64 @@
-configuration: Debug
-image: Visual Studio 2019
-
-version: '{build}'
-
-init:
-- ps: Update-AppveyorBuild -Version "$($env:ospsuite_version).$($env:appveyor_build_version)"
-
-environment:
-  MIK_TEX_INSTALL_DIR: 'C:\MikTex\'
-
-dotnet_csproj:
-  patch: true
-  file: '**\*.csproj'
-  version: '$(ospsuite_version).$(build_number)'
-  assembly_version: '$(ospsuite_version).$(build_number)'
-  file_version: '$(ospsuite_version).$(build_number)'
-  informational_version: '$(ospsuite_version).$(build_number)'
-
-clone_depth: 1
-
-install:
-- set PATH=C:\Ruby25\bin;%PATH%
-- gem install xml-simple
-- ps: >-
-    (new-object net.webclient).DownloadFile('https://github.com/Open-Systems-Pharmacology/MiKTeX/releases/download/v2.9.3/MikTex.2.9.3.msi', 'C:\MikTex.msi')
-
-    Start-Process msiexec.exe -ArgumentList /i, C:\MikTex.msi, /quiet, /qn, /norestart, INSTALLDIR=$env:MIK_TEX_INSTALL_DIR -Wait
-
-before_build:
-  - nuget sources add -name bddhelper -source https://ci.appveyor.com/nuget/ospsuite-bddhelper 
-  - nuget sources add -name utility -source https://ci.appveyor.com/nuget/ospsuite-utility 
-  - nuget sources add -name serializer -source https://ci.appveyor.com/nuget/ospsuite-serializer 
-  - nuget sources add -name texreporting -source https://ci.appveyor.com/nuget/ospsuite-texreporting 
-  - nuget sources add -name databinding -source https://ci.appveyor.com/nuget/ospsuite-databinding 
-  - nuget sources add -name databinding-devexpress -source https://ci.appveyor.com/nuget/ospsuite-databinding-devexpress 
-  - nuget sources add -name funcparser -source https://ci.appveyor.com/nuget/ospsuite-funcparser 
-  - nuget sources add -name simmodel -source https://ci.appveyor.com/nuget/ospsuite-simmodel
-  - nuget sources add -name cvodes -source https://ci.appveyor.com/nuget/ospsuite-simmodel-solver-cvodes
-  - nuget sources add -name core -source https://ci.appveyor.com/nuget/ospsuite-core
-  - nuget restore
-  - rake "update_go_license[ApplicationStartup.cs, %GO_DIAGRAM_KEY%]"
-  
-after_build:
-  - rake "create_setup[%APPVEYOR_BUILD_VERSION%, %CONFIGURATION%]"
-  - 7z a setup.zip %APPVEYOR_BUILD_FOLDER%\setup\deploy\*.msi
-  - rake "create_portable_setup[%APPVEYOR_BUILD_VERSION%, %CONFIGURATION%, mobi-portable-setup.zip]"
-
-build:
-  verbosity: minimal
-  project: MoBi.sln
-
-test:
-  assemblies:
-    - MoBi.Tests.dll
-    - MoBi.UI.Tests.dll
-
-on_finish: 
-  - ps: Get-ChildItem setup.zip | % { Push-AppveyorArtifact $_.FullName -FileName $_.Name }
-  - ps: Get-ChildItem .\setup\*.zip | % { Push-AppveyorArtifact $_.FullName -FileName $_.Name }
-
-skip_commits:
-  files:
+configuration: Debug
+image: Visual Studio 2019
+
+version: '{build}'
+
+init:
+- ps: Update-AppveyorBuild -Version "11.3.$($env:appveyor_build_version)"

```

Comment:
wondering if there is not a better way of doing this
e.g. define a variable in the file that points to this value and then use it. No idea. Anyways good for now